src/rpc/blockchain.cpp

```diff
@@ -552,6 +552,32 @@ UniValue MempoolToJSON(const CTxMemPool& pool, bool verbose, bool include_mempoo
     }
 }
 
+static RPCHelpMan maxmempool()
+{
+    return RPCHelpMan{"maxmempool",
```

Comment:
Not sure about concept ACK/NACK here myself yet, but better naming would be `setmaxmempool` IMO.